Data/ByteString/Lazy.hs

```diff
@@ -300,7 +301,7 @@ null _     = False
 -- | /O(c)/ 'length' returns the length of a ByteString as an 'Int64'
 length :: ByteString -> Int64
 length cs = foldlChunks (\n c -> n + fromIntegral (S.length c)) 0 cs
-{-# INLINE length #-}
+{-# INLINE [1] length #-}
```

Comment:
I added `{-# INLINE [1] length #-}`, because without it the compiler might inline `length` before applying the `compareLength` rules.